CMakeLists.txt

```diff
@@ -311,7 +314,7 @@ endif()
 # FIXME: Support ARM "-mfpu=vfp -mfloat-abi=hard"
 if (("${TARGET_ARCHITECTURE}" STREQUAL "x86_64") OR ("${TARGET_ARCHITECTURE}" STREQUAL "i686"))
   if (("${CMAKE_CXX_COMPILER_ID}" MATCHES "GNU") OR ("${CMAKE_CXX_COMPILER_ID}" MATCHES "Clang"))
-    set(SSE_FLAGS "-mfpmath=sse" "-msse" "-msse2")
+    set(SSE_FLAGS "-mfpmath=sse" "-msse" "-msse2" "-std=c++0x")
```

Comment:
Yes it is a breaking change. C++11 was tested in Lev's branch for some time on a reasonable slice of Z3.
The C++11 directive to the master branch was added Saturday by Wintersteiger. One observation was that the gcc build time increased and our builds started hitting a timeout.